src/components/PeoplePage.tsx

```diff
@@ -1,8 +1,91 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Param } from '../types/Param';
+
+const sortingPeople = (list: Person[], sorter: Param, order?: string) => {
+  return list.sort((a: Person, b: Person) => {
+    if (order?.length === 0) {
+      switch (sorter) {
+        case 'name':
+          return a.name.localeCompare(b.name);
+        case 'sex':
+          return a.sex.localeCompare(b.sex);
```

Comment:
```suggestion
        case 'name':
        case 'sex':
          return a[sorter].localeCompare(b[sorter]);
```
you can group the cases